osmosis-1/osmosis-1.assetlist.json

```diff
@@ -17192,6 +17192,63 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/COCA.png"
       }
+    },
+    {
```

Comment:
Please do not add to this file--it will soon be deleted.

Reply:
reverted this